config/initializers/govuk_error.rb

```diff
@@ -3,8 +3,9 @@
 
   # Don't capture postgres errors that occur during the time that the data sync
   # is running in integration and staging environments
-  config.should_capture = ->(error) do
-    data_sync_ignored_error = error.is_a?(PG::Error) || error.cause.is_a?(PG::Error)
+  config.should_capture = ->(error_or_event) do
```

Comment:
I think it would be helpful to extend the comment a little, to reinforce the possible data types.

Reply:
That's a good idea. I've done so.